src/index.html

```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <div class="from-field">
+          <label
+            >Surname:
+            <input type="text"
+                   name="surname"
+                   autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >Name:
+            <input type="text"
+                   name="name"
+                   autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >How old are You?
+            <input type="number"
+                   name="age"
+                   value="12"
+                   min="1"
+                   max="100"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >Full date of birth
+            <input type="date"
+                   name="dateOfBirth"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >I accept the term of agreement
+            <input type="checkbox"
+                   name="terms"
+                   required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section">
+        <legend>Registration</legend>
+        <div class="from-field">
+          <label
+            >E-mail:
+            <input type="email"
+                    name="email"
+                    required
+                    placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div class="from-field">
+          <label
+            >Password:
+            <input type="password"
+                   name="password"
+                   minlength="8"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section">
+        <legend>An interesting fact about you</legend>
+        <div class="from-field">
+          Do you love cats?
+          <label
+            >Yes
+            <input type="radio"
+                   name="email"
```

Comment:
The 'name' attribute for the 'Yes' radio input should be unique, e.g., 'loveCats', instead of 'email'.